lib/market.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+class Market
+  attr_reader :name, :vendors
+  def initialize(name, vendors = [])
+    @name = name
+    @vendors = vendors
+    @names = []
```

Comment:
You don't need to store names; this data is already on the vendors in your `@vendors` array.